src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/extensions/arquillian/ConfigureArquillianTask.groovy

```diff
@@ -1,27 +1,42 @@
 package net.wasdev.wlp.gradle.plugins.tasks.extensions.arquillian;
 
+import org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler
+import org.gradle.api.internal.tasks.TaskDependencyInternal
+import org.w3c.dom.Element
+import net.wasdev.wlp.gradle.plugins.utils.LooseWarApplication
+import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.artifacts.dsl.DependencyHandler
+import org.apache.commons.io.FileUtils;
 import java.io.File
 import java.io.FileNotFoundException
 import java.io.IOException
 import java.util.Map.Entry
-
 import javax.xml.parsers.ParserConfigurationException
 import javax.xml.xpath.XPathExpressionException
-
 import org.gradle.api.GradleException
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskAction
+import org.gradle.internal.impldep.bsh.This
 import org.xml.sax.SAXException
-
+import net.wasdev.wlp.common.arquillian.objects.LibertyProperty;
+import net.wasdev.wlp.common.arquillian.objects.LibertyRemoteObject
 import net.wasdev.wlp.common.arquillian.objects.LibertyManagedObject
 import net.wasdev.wlp.common.arquillian.objects.LibertyProperty
 import net.wasdev.wlp.common.arquillian.util.ArquillianConfigurationException
 import net.wasdev.wlp.common.arquillian.util.Constants
 import net.wasdev.wlp.common.arquillian.util.HttpPortUtil
 import net.wasdev.wlp.gradle.plugins.tasks.AbstractServerTask
+import net.wasdev.wlp.gradle.plugins.tasks.extensions.arquillian.ConfigureArquillianTask.TypeProperty
```

Comment:
I don't think we need this as the enum is inside the class already.

Reply:
What do you mean by this?